scripts/builder/timestamp_reset.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+set -e
+
+RESET_TIMESTAMP="2001-01-01T01:01:01"
+
+if [[ -d ${FILEPATH} ]]; then
+    # find ./${FILEPATH} -type f -exec "touch -d ${RESET_TIMESTAMP}" {} +
```

Comment:
Assume this is left over from testing the command, do we need this?

Reply:
whoopsy! will remove

Reply:
fixed